specifications/transfer-claim/transfer-claim.yml

```diff
@@ -70,6 +70,35 @@ paths:
       security:
       - basic_auth: []
 
+  /creditor/{creditorId}/settlement/{payoutReference}:
+    get:
+      tags:
+        - Creditor endpoints
+      summary: Get settlement details for payout
+      description: Used to fetch the details of a payout to the creditor.
```

Comment:
Does this sound better?
> Used to fetch the details of a payment received by the creditor.

Reply:
Since we have `payout` everywhere else, I'm not sure.
And if we change everywhere to "payment" then it becomes confusing between the "paymentReference" and "PayoutReference"
![image](https://github.com/Moank/moank-open-api-docs/assets/57348006/e8331353-e9e3-4165-a040-e8558675d65e)